platform-sdk/swirlds-metrics-api/src/main/java/com/swirlds/metrics/api/Metrics.java

```diff
@@ -171,4 +171,12 @@ default void resetAll() {
      * 		if {@code updater} is {@code null}
      */
     void removeUpdater(final @NonNull Runnable updater);
+
+    /**
+     * Shut down the service.
+     *
+     * @return {@code true} if the shutdown finished on time, {@code false} if the call ran into a timeout
+     * @throws InterruptedException if the current thread was interrupted while waiting
+     */
+    boolean shutdown() throws InterruptedException;
```

Comment:
I thought about it some more. I do not think we want a `shutdown()` method on `Metrics`.

Reply:
I wanted to have a more elegant way to clear the metrics agnostic of the metrics type but this method seems to not be applicable for each type of Metrics. I removed it.